Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -282,25 +273,22 @@ install_package "ethtool"
 
 			# Check the system log if it shows Power Management log
 			"hibernation entry", "hibernation exit" | ForEach-Object {
-				# TODO: For other distro, need to check out the syslog or messages.
-				$pm_log_filter = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "cat /var/log/syslog | grep -i '$_'" -ignoreLinuxExitCode:$true
+				if (($global:detectedDistro -imatch "CENTOS") -or ($global:detectedDistro -imatch "REDHAT") ) {
+					$pm_log_filter = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "dmesg | grep -i '$_'" -runAsSudo -ignoreLinuxExitCode:$true
+				} else {
+					$pm_log_filter = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "cat /var/log/syslog | grep -i '$_'" -runAsSudo -ignoreLinuxExitCode:$true
+				}
```

Comment:
The same comment, do we need "-runAsSudo".

Reply:
I post the answer above comment